beetmoverscript/utils.py

```diff
@@ -119,7 +119,6 @@ def generate_beetmover_template_args(context):
         "branch": release_props["branch"],
         "product": release_props["appName"],
         "stage_platform": release_props["stage_platform"],
-        "platform": release_props["platform"],
```

Comment:
Hm, I don't think this will work. Removing `platform` will cause templates to fail. There's plenty of references to "{{ platform }} in all over the templates from https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/templates/ (for example https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/templates/firefox_candidates.yml#L13 )

This is another reason why I should've added integration tests to this repo long time ago, bah.
@lisajguo can you please try to run beetmover locally and confirm this works or not? Something similar to what we've done together on Monday. Please test with a `beetmover-repackage` and `beetmover-checksums` beetmover jobs.

Reply:
Yep, on it